app/services/subcase-is-approved.js

```diff
@@ -28,4 +28,40 @@ export default class SubcaseIsApprovedService extends Service {
     }
     return false;
   }
+
+  async isRetracted(subcase) {
+    const meeting = await subcase?.requestedForMeeting;
+
+    if (meeting?.isFinal) {
+      const retractedDecisionResultCode = await this.store.findRecordByUri(
+        'decision-result-code',
+        CONSTANTS.DECISION_RESULT_CODE_URIS.INGETROKKEN
+      );
+
+      const nrDecisionActivities = await this.store.count('decision-activity', {
+        'filter[subcase][:id:]': subcase.id,
+        'filter[decision-result-code][:id:]': retractedDecisionResultCode.id,
+      });
+      return nrDecisionActivities > 0;
+    }
+    return false;
+  }
+
+  async isPostponed(subcase) {
+    const meeting = await subcase?.requestedForMeeting;
+
+    if (meeting?.isFinal) {
+      const postponedDecisionResultCode = await this.store.findRecordByUri(
+        'decision-result-code',
+        CONSTANTS.DECISION_RESULT_CODE_URIS.UITGESTELD
+      );
+
+      const nrDecisionActivities = await this.store.count('decision-activity', {
+        'filter[subcase][:id:]': subcase.id,
+        'filter[decision-result-code][:id:]': postponedDecisionResultCode.id,
+      });
+      return nrDecisionActivities > 0;
+    }
+    return false;
```

Comment:
Is it correct to consider a subcase only being postponed if the meeting is finalized? (same question for retraction)